myhoard/web_server.py

```diff
@@ -133,6 +133,8 @@ async def status_update(self, request):
             elif body_mode == Controller.Mode.observe:
                 self.controller.switch_to_observe_mode()
             elif body_mode == Controller.Mode.restore:
+                if not isinstance(body.get("rebuild_tables"), (bool, type(None))):
```

Comment:
If I am reading this correctly, in case of body containing:
```
rebuild_tables: null
```
It will pass None to the restore_backup which is not expected, as it's not optional, even though I assume the code will work.